e2e/configuration_test.go

```diff
@@ -124,25 +123,25 @@ continueCheck:
 	}
 
 	klog.Info("6. Checking Secrets and ConfigMap which should all be deleted")
-	_, err = clientSet.CoreV1().Secrets("default").Get(ctx, "variable-alibaba-eip", v1.GetOptions{})
+	_, err = clientSet.CoreV1().Secrets("default").Get(ctx, "variable-random-e2e", v1.GetOptions{})
 	assert.Equal(t, kerrors.IsNotFound(err), true)
 
-	_, err = clientSet.CoreV1().Secrets("default").Get(ctx, "eip-conn", v1.GetOptions{})
+	_, err = clientSet.CoreV1().Secrets("default").Get(ctx, "random-conn", v1.GetOptions{})
 	assert.Equal(t, kerrors.IsNotFound(err), true)
 
-	_, err = clientSet.CoreV1().Secrets(backendSecretNamespace).Get(ctx, "tfstate-default-alibaba-eip", v1.GetOptions{})
+	_, err = clientSet.CoreV1().Secrets(backendSecretNamespace).Get(ctx, "tfstate-default-random-e2e", v1.GetOptions{})
 	assert.Equal(t, kerrors.IsNotFound(err), true)
 
-	_, err = clientSet.CoreV1().ConfigMaps("default").Get(ctx, "tf-alibaba-eip", v1.GetOptions{})
+	_, err = clientSet.CoreV1().ConfigMaps("default").Get(ctx, "tf-random-e2e", v1.GetOptions{})
 	assert.Equal(t, kerrors.IsNotFound(err), true)
 }
 
-func TestBasicConfigurationRegression(t *testing.T) {
-	var retryTimes = 120
-
-	klog.Info("0. Create namespace")
-	err := exec.Command("bash", "-c", "kubectl create ns abc").Start()
-	assert.NilError(t, err)
-
-	Regression(t, testConfigurationsRegression, retryTimes)
-}
+//func TestBasicConfigurationRegression(t *testing.T) {
+//	var retryTimes = 120
+//
+//	klog.Info("0. Create namespace")
+//	err := exec.Command("bash", "-c", "kubectl create ns abc").Start()
+//	assert.NilError(t, err)
+//
+//	Regression(t, testConfigurationsRegression, retryTimes)
+//}
```

Comment:
Will these part of codes be removed permanently?

Reply:
Yes, it should keep until I find a solution. The block is for the regression test which has been running for a few months. But here is a security risk as the secret of a Terraform Provider can be literally printed in the action. So I commented it for now.